src/js/game/Entities/Player.js

```diff
@@ -31,7 +32,7 @@ module.exports = class Player extends BaseEntity {
     const timeEllapsed = (+new Date() - this.lastMovement);
     const movementAlmostFinished = timeEllapsed > 300;
 
-    if ((queueIsEmpty || (queueHasOne && movementAlmostFinished)) && isMoving) {
+    if (!this.onTracks && ((queueIsEmpty || (queueHasOne && movementAlmostFinished)) && isMoving)) {
```

Comment:
can we pull this out into a "canUpdateMovement" method?

Reply:
👍